test/types/tuple/types/testTuple.bad

```diff
@@ -0,0 +1 @@
+testTuple.chpl:8: error: modules (like 'testTuple' here) cannot be called like procedures
```

Comment:
This one seems strange.

Reply:
Agreed. Upon closer inspection, this was a test error (test name matched the procedure name). Pushing an update soon.